modules/tasks/tasks/src/main/java/org/motechproject/tasks/ex/ProviderNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package org.motechproject.tasks.ex;
+
+/**
+ * Signals that provider with a given id was not found.
+ */
+public class ProviderNotFoundException extends IllegalArgumentException {
+
```

Comment:
I doubt we will be serializing this, but I guess I can add it for the sake of removing an IDE warning